File: integrations/php/oicana-php/src/Manifest/FontConfig.php
<?php

declare(strict_types=1);

namespace Oicana\Manifest;

/**
 * Fonts a template expects from its host.
 */
final readonly class FontConfig
{
    /**
     * @param list<string> $require Font families the host has to register
     */
    public function __construct(
        public array $require
    ) {
    }
}

File: integrations/csharp/Oicana/Manifest/FontConfig.cs
namespace Oicana.Manifest;

/// <summary>
/// Fonts a template expects from its host.
/// </summary>
public sealed class FontConfig
{
    /// <summary>
    /// Font families the host has to register.
    /// </summary>
    public required IReadOnlyList<string> Require { get; init; }
}
